plugins/services/src/js/reducers/serviceForm/Container.js

```diff
@@ -245,20 +245,23 @@ module.exports = {
       this.isMesosRuntime = value === MESOS;
     }
 
-    const newState = Object.assign(
+    let newState = Object.assign(
       {},
       containerJSONReducer.apply(this, [this.internalState, ...args])
     );
 
     this.internalState = newState;
+    if (type === SET && path.join(".") === "container") {
```

Comment:
That should do. When switching to multi-container we're emit `SET` transaction with `value == undefined` to delete the `container` field.  This results in the exception.

```suggestion
    if (type === SET && path.join(".") === "container" && value) {
```